corva/models/stream.py

```diff
@@ -63,14 +78,52 @@ def is_completed(self) -> bool:
         return False
 
 
-class StreamEvent(ListEvent[StreamEventData]):
+class StreamEvent(BaseEvent, StreamEventData):
+    @staticmethod
+    def from_raw_event(event: str, **kwargs) -> List[StreamEvent]:
+        app_key = kwargs['app_key']
+
+        events = parse_raw_as(List[StreamEvent], event)  # type: List[StreamEvent]
+
+        for event in events:
+            event.app_key = app_key
+
+        return events
+
     @staticmethod
-    def from_raw_event(event: str, **kwargs) -> StreamEvent:
-        app_key: str = kwargs['app_key']
+    def filter(
+         event: StreamEvent, by_timestamp: bool, by_depth: bool, last_timestamp: int, last_depth: float
+    ) -> StreamEvent:
+        records = event.records
+
+        if event.is_completed:
+            records = records[:-1]  # remove "completed" record
```

Comment:
we can have only 1 completed record and it will be always at the end of the list?

Reply:
Yes

Reply:
probably need to document that here, because it is not clear when reading this code